components/datetime/src/format/neo.rs

```diff
@@ -1809,6 +1829,28 @@ impl<R: DateTimeNamesMarker> RawDateTimeNames<R> {
         Ok(())
     }
 
+    pub(crate) fn load_time_zone_primaries<P>(
+        &mut self,
+        provider: &P,
+    ) -> Result<(), SingleLoadError>
+    where
+        P: BoundDataProvider<tz::PrimaryZonesV1Marker> + ?Sized,
+    {
+        self.primary_zones
+            .load_put(provider, Default::default(), ())
+            .map_err(|e| {
+                MaybePayloadError2::into_single_load_error(
+                    e,
+                    fields::Field {
+                        symbol: FieldSymbol::TimeZone(fields::TimeZone::UpperV),
+                        length: FieldLength::Wide,
```

Comment:
Thought: we should probably plumb the NeoSkeletonLength through to here. (seems minor enough to make it an issue to be fixed later)